WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1814,6 +1815,27 @@ private void refreshBlogMedia() {
                 @Override
                 public void onSuccess(int count) {
                     mBlogMediaStatus = 1;
+
+                    runOnUiThread(new Runnable() {
+                        @Override
+                        public void run() {
+                            if (!mPendingVideoPressInfoRequests.isEmpty()) {
+                                // If there are pending requests for video URLs from VideoPress ids, query the DB for
+                                // them again and notify the editor
+                                for (String videoId : mPendingVideoPressInfoRequests) {
+                                    String blogId = String.valueOf(WordPress.currentBlog.getLocalTableBlogId());
```

Comment:
Minor nit: we could move the `blogId` assignment above the `for` loop since it'll be the same for all pending videos.